disco/extensions/upgrade_simulation/upgrades/thermal_upgrade_functions.py

```diff
@@ -328,6 +357,10 @@ def identify_parallel_xfmrs(options=None, row=None, parallel_xfmrs_limit=None):
     num_parallel_xfmrs = int(chosen_option["num_parallel"])
     chosen_option["conns"] = convert_list_string_to_list(chosen_option["conns"])
     chosen_option["kVs"] = convert_list_string_to_list(chosen_option["kVs"])
+    if isinstance(chosen_option["%Rs"], str):
+                    chosen_option["%Rs"] = convert_list_string_to_list(chosen_option["%Rs"])

```

Comment:
Is the function misnamed? `convert_string_to_list`?

Reply:
I couldnt think of a better name-- this basically converts "[2,3,4]" to [2,3,4].
Hence the name "list_string" to "list" -- Looks like thats confusing. I guess we could just call it this then : convert_string_to_list

Reply:
Ah. Consider `ast.literal_eval()`

Reply:
Oh I just checked -- ast.literal_eval is what I used inside that function :D 
I guess I made that function initially with something else in. Replaced with ast.literal_eval in all usages